src/proofs.rkt

```diff
@@ -12,8 +12,7 @@
 (define (box-set stx components ctx)
   (let loop ([stx stx])
     (match stx
```

Comment:
Note that this function no longer uses the `components` argument ever. So you can delete it here and in all of the callers.